docs/docs/workspace-settings/README.md

```diff
@@ -0,0 +1,93 @@
+# Workspace Settings
+
+When you sign up for a Pipedream account, you'll be guided to create a workspace. After you create your workspace, your workflows and projects will be assigned to your private workspace.
+
+A workspace is a collection of **projects**, **folders** and **workflows**. You can develop workflows within a private workspace, or you can invite others to join your workspace.
+
+You can be a member of one or more workspaces.
+
+## Creating a new workspace
+
+To create another workspace, open the dropdown menu in the top left of the Pipedream dashboard.
+
+Then select **Create new workspace**.
+
+You'll be prompted to name the workspace, but don't worry you can [change the name later](/workspaces/#renaming-a-workspace).
```

Comment:
"You'll be prompted to name the workspace. You can [change the name later](/workspaces/#renaming-a-workspace)."

Reply:
Agreed